api_app/playbooks_manager/controller.py

```diff
@@ -0,0 +1,150 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import logging
+from typing import Dict, List, Tuple
+
+from celery import chord
+from django.conf import settings
+
+from api_app.exceptions import (
+    NotRunnablePlaybook,
+)
+from ..utility import job_cleanup, runnable_connectors, stack_analyzers, stack_connectors, runnable_analyzers
+
+from ..models import Job
+from .dataclasses import PlaybookConfig
+
+logger = logging.getLogger(__name__)
+
+def filter_playbooks(serialized_data: Dict) -> Tuple[List[str]]:
```

Comment:
Wrong signature

Reply:
I will just leave it to be a Tuple of Lists then?